pkg/network/helper.go

```diff
@@ -2,7 +2,6 @@ package network
 
 type Helper struct {
 	EventSender func(event *Event, networkType string) error
-	Resetter    func(networkType string) error
 }
```

Comment:
The Helper type seems unnecessary. can it be something like
```
type EventSender interface{
  func(event *Event, networkType string) error
}
```

Reply:
Yes, it seems that interface is better than structure. I reserve the name `helper` and reimplement it as an interface instead of structure.

Reply:
Help is too generic as an interface name so I suggest we change that